app/config/config.go

```diff
@@ -0,0 +1,42 @@
+// Package config works with config file for building pages
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+// Domain struct is data about domain what we neet to evaluate
+type Domain struct {
+	Name        string `json:"name"`
+	PricePerDay int    `json:"price_per_day,omitempty"`
+	Rate        int    `json:"rate,omitempty"`
+}
+
+// Config struct is data about domais
+// and common values for domains
+// without overrided PricePerDay and Rate
+type Config struct {
+	Domains     []Domain `json:"domains"`
+	PricePerDay int      `json:"price_per_day"`
+	Rate        int      `json:"rate"`
+}
+
+// ReadConfig returns content of file
+// which was readed by provided filepath
+func ReadConfig(filepath string) Config {
+	fmt.Printf("Read config file: %q\n", filepath)
```

Comment:
I would change it to `log` because it would give you better control over format of the ouput, which is not available to you with `fmt`.